### ISSUE TITLE
Virtual Controller is always connected when rainway is running

### ISSUE THREAD

**Issue by DjScribbles** [RainwayApp/bug-tracker#220]
The Rainway Virtual Controller device is connected as soon as Rainway starts up, rather than when the a stream begins. This creates issues with key-prompts in a multitude of games that display xbox prompts any time a controller is detected, and can create more serious issues in games like Skyrim, where having a new controller present at startup defaults you to exclusive-controller mode (ignoring all Keyboard-mouse input until you navigate the menus with the controller to switch back), which would make the game unplayable if you don't have a controller.

Adding the virtual device when the stream starts would probably create a new issue in some games that can't detect new controllers after startup, if the game was already running when the stream connected. This could be resolved by providing an option to restart the application when starting the stream.

If the trade-off for the fix is debatable, having an option to select whether to create the controller at startup, or when beginning the stream may be a happy middle-ground.

**Comment by DeLaguna** [comment]
I would say connect controller when the remote device initiates its connection with Railway, and the remote device is looking at the game list. Because that is when you are preparing to play games, and games shouldn't launch until the controller is connected, if not connected then pop up prompt allowing force start but with a warning that game may not be playable. if no input or change to the game then crashes back to the choice screen and prompt troubleshooting. .... just my two cents.
